README.md

```diff
@@ -10,6 +10,14 @@ It will eventually have MIDI input, and a way to share music and instruments wit
 
 [Test it out here!](https://fazeoid.netlify.app/)
 
+Supported browsers:
+ - Chrome desktop
+ - Chrome mobile
```

Comment:
Should rename this to Chrome Android because iOS does not use the same rendering engine iirc. As a matter of fact, ALL ios browsers use safari to render.